Makefile

```diff
@@ -11,5 +11,6 @@ theme: mtheme
 
 clean:
 	latexmk -c
+	rm -rf _minted-slides/ *.sty *.snm *.vrb *.nav
```

Comment:
Hm, are you sure it's necessary to clean the theme files (*.sty)? (OTOH they're regenerated when using `make` again, so I guess it's fine...)